flaskr/auth.py

```diff
@@ -12,12 +12,16 @@
 bp = Blueprint('auth', __name__, url_prefix='/auth')
 
 # associates the URL `/register` with the register view function below
-@bp.route('/register', methods=('GET', 'POST'))
+@bp.route('/', methods=('GET', 'POST'))
```

Comment:
Why are you removing the /register endpoint here?

Reply:
I did that because there was no index.html, so now if you just go to auth/ it will show the register page